<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="View" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>460</height>
    </rect>
   </property>
   <property name="renderHints">
    <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
   </property>
   <property name="optimizationFlags">
    <set>QGraphicsView::IndirectPainting</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>100</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="addButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>add point</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="editButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>edit points</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showPoints">
       <property name="text">
        <string>show points</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showLines">
       <property name="text">
        <string>show lines</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>140</y>
     <width>100</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Animation</string>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="duckButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Duck</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sharkButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Shark</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QGraphicsView</extends>
   <header>view.h</header>
   <slots>
    <slot>enableEditMode(bool)</slot>
    <slot>showPoints(bool)</slot>
    <slot>sharkToDuck()</slot>
    <slot>duckToShark()</slot>
    <slot>showLines(bool)</slot>
    <slot>save()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>editButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>graphicsView</receiver>
   <slot>enableEditMode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>duckButton</sender>
   <signal>clicked()</signal>
   <receiver>graphicsView</receiver>
   <slot>sharkToDuck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showPoints</sender>
   <signal>clicked(bool)</signal>
   <receiver>graphicsView</receiver>
   <slot>showPoints(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sharkButton</sender>
   <signal>clicked()</signal>
   <receiver>graphicsView</receiver>
   <slot>duckToShark()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showLines</sender>
   <signal>clicked(bool)</signal>
   <receiver>graphicsView</receiver>
   <slot>showLines(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
